content/induction/sprints/1/prep/index.md

```diff
@@ -0,0 +1,353 @@
+Explain the difference between Git and Github
+Get a local copy of a repository on your local machine
+
+## 📝 CYF blog
+
+Imagine a scenario where 2 people are writing a blog.
+Ahmed and Naima are co-authoring a blog to document the things they're struggling with whilst learning to code.
+
+The published blog is live here 👉 [CYF educational blog](https://git-demo-week1.netlify.app/)
+
+{{<note type="exercise" title="exercise 1.1">}}
+Try exploring the site above. Try clicking on the link on the site and check to see if its working.
+{{</note>}}
+
+## 🧰 Development process
+
+Ahmed and Naima are currently using the following **development process** for writing the blog:
+
+> - writing the blog in a single file on a **single** computer
+> - saving multiple versions of the file on the same computer
+> - they take it in turns to the use the computer at different points in the day
+
+At the moment, the computer has a folder with the blog that looks like this:
+
+![different-blog-versions](different-blog-versions.png)
+
+{{<note type="exercise" title="exercise 1.2">}}
+
+Describe some of the challenges that Ahmed and Naima face when trying to write a blog together in this way.
+
+Try creating a thread 🧵 Slack to see if your cohort colleagues came up with similar answers.
+
+{{</note>}}
+
+## ⌛ Version control software
+
+> To improve their way of working, Ahmed and Naima realise they need the following:
+>
+> - a way to create and manage the different versions of their project
+> - a way to share the history of the project between different users
+
+To manage the different versions of the blog project, they decide to use **Git**.
+
+{{<note type="definition" title="definition">}}
+
+Git is **version control software** that allows developers to create and manage different versions of a project.
+
+{{</note>}}
+
+In Git, we create different versions of a project over time by creating a **commit**.
+
+A **commit** is a snapshot of our project at a particular point in time. You can also think of a **commit** as a particular version of a project.
+
+**Commits** will give us the following information👍
+
+- what changed in this commit
+- who created the change
+- what time the change happen
+- what the previous commit was
+
+A typical timeline of commits might look like this:
+
+![commit-history](commit-history.png)
+
+## 🗄️ 🌐 Sharing history
+
+Earlier on, Ahmed and Naima realised they also need the following:
+
+> a way to share the history of the project between different users
+
+To share a project and its history, we can use an online platform called **Github**
+
+**Github** is a platform where teams can store projects along with a history of their different versions.
+
+By storing projects on Github, multiple users can gain access to the history of a project.
+
+{{<note type="definition" title="Definition">}}
+On Github we call our project and its history a **repository**.
+{{</note>}}
+
+{{<note type="exercise" title="Exercise 2.1">}}
```

Comment:
Could we have more meaningful titles please? To create a better navigation experience.